assets/panel/debugger.properties

```diff
@@ -271,6 +271,8 @@ breakpointMenuItem.deleteAll2.label=Remove all
 breakpointMenuItem.deleteAll2.accesskey=a
 breakpointMenuItem.removeCondition2.label=Remove condition
 breakpointMenuItem.removeCondition2.accesskey=c
+breakpointMenuItem.addCondition2.label=Add condition
```

Comment:
Why addCondition2 and not just addCondition?

Reply:
I didn't know why the others got renamed I I just wanted to ensure consistency. Should I remove the "2" from my new entries @flodolo

Reply:
I'd keep the 2 to be consistent. We recently changed the labels to drop "breakpoint" e.g. `Remove breakpoint condition` to `Remove condition`...